server/modules/workflows/rest/index.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express'
+import isAuthorizedWithSpeckle from '../services/isAuthorizedWithSpeckle.js'
+
+const router = Router()
+
+router.use(isAuthorizedWithSpeckle)
```

Comment:
I thought I could piggyback on Speckle's authorisation for the server's API. If the Bearer token resolves to a user ID, then that user is authenticated. I can then use the ID to retrieve their workflows etc. from the db. @teocomi @didimitrie 

@didimitrie @teocomi